.husky/commit-msg

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
Is this the code that is generated by husky? I've used husky in the past but this is new to me.